src/backend/internal/model/model.go

```diff
@@ -41,6 +51,18 @@ type UserInput struct {
 	Lastname  string  `json:"lastname" example:"test"`
 }
 
+type UserUpdateInput struct {
+	Email       *string `json:"email" example:"test@mail.com"`
+	Password    *string `json:"password" example:"root"`
+	PhoneNumber *string `json:"phone_number" example:"096198923"`
+	Firstname   *string `json:"firstname" example:"test"`
+	Lastname    *string `json:"lastname" example:"test"`
+	Gender      *string `json:"gender" example:"Male"`
+	About       *string `json:"about" example:"Hello"`
+	Username    *string `json:"username" example:"test"`
+	Location    *string `bun:"name,type:varchar" json:"location"`
```

Comment:
Possible a typo kub.

`json:"address" example:"Bangkok"` if you have made changes according to the previous comment.